bin/storm.py

```diff
@@ -754,18 +827,24 @@ def parse_config_opts(args):
     elif token == "--config":
       global CONFFILE
       CONFFILE = curr.pop()
+    elif token == "--jars":
```

Comment:
Are these two options common for commands ? If so, could we have some help info in the `print_command` function ?